cgmanifests/generated/cgmanifest.json

```diff
@@ -78,56 +78,6 @@
         "comments": "manylinux dependency LIBXCRYPT"
       }
     },
-    {
```

Comment:
They should not be deleted. They were missed because your local folder didn't have the submodules. I should add this to our document.  No worries. I will fix it in a later commit.